public-ui/src/app/views/ehr-view/ehr-view.component.ts

```diff
@@ -188,6 +189,14 @@ export class EhrViewComponent implements OnInit, OnDestroy {
   }
 
   public searchDomain(query: string) {
+    if (query) {
+      window['dataLayer'].push({
+        'event': 'domainPageSearch',
+        'category': 'Search',
+        'action': 'Search Inside Domain' + ' ' + this.ehrDomain.name,
```

Comment:
nit: put the space inside the first string instead of concat'ing

Reply:
Done thanks.